app/views/step_by_step_pages/internal_change_notes.html.erb

```diff
@@ -56,7 +56,7 @@
             <%= note.readable_created_date %>
           </summary>
           <p>Change made by <strong><%= note.author %></strong></p>
-          <pre><%= note.description %></pre>
+          <pre class="change-note--description"><%= note.description %></pre>
```

Comment:
What does this change do?

Reply:
According to [w3schools](https://www.w3schools.com/CSSref/pr_text_white-space.asp) it "Whitespace is preserved by the browser. Text will wrap when necessary, and on line breaks"